service/src/main/java/eu/europa/ec/fisheries/ers/service/facatch/evolution/EntryCatchProgressHandler.java

```diff
@@ -0,0 +1,23 @@
+package eu.europa.ec.fisheries.ers.service.facatch.evolution;
+
+import eu.europa.ec.fisheries.ers.fa.entities.FaCatchEntity;
+import eu.europa.ec.fisheries.ers.fa.entities.FishingActivityEntity;
+import eu.europa.ec.fisheries.ers.fa.utils.FaReportDocumentType;
+import eu.europa.ec.fisheries.ers.service.dto.fishingtrip.catchprogress.CatchProgressDTO;
+import eu.europa.ec.fisheries.ers.service.dto.fishingtrip.catchprogress.EntryCatchProgressDTO;
+
+public class EntryCatchProgressHandler extends CatchProgressHandler {
```

Comment:
See my comments on DepCatchProgressHandler class, you should apply the same logic here